firefox-ios/firefox-ios-tests/Tests/ClientTests/Search/TrendingSearchClientTests.swift

```diff
@@ -30,9 +30,6 @@ final class TrendingSearchClientTest: XCTestCase {
             "golden retriever tricks",
             "best travel destinations 2025",
             "cozy coffee shops",
-            "sleepy kittens",
-            "board game night ideas",
-            "dogs wearing sunglasses"
```

Comment:
can't believe we're getting rid of dogs wearing sunglasses. SHAME!

Reply:
you ask and you shall receive!!!